code/src/components/CheckBox.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+const CheckBox = (props) => {
+    const {nextQuestion, setPersonality, personality, setPerson, person } = props
```

Comment:
could be simplified by writing out the consts instead of props like this -  const checkbox = ({nextQuestion, setPersonality, personality, setPerson, person })